apps/web/public/static/locales/en/common.json

```diff
@@ -861,12 +861,17 @@
   "copy_private_link": "Copy private link",
   "copy_private_link_to_event": "Copy private link to event",
   "private_link_description": "Generate a private URL to share without exposing your {{appName}} username",
+  "single_use_links_title": "Single Use Links",
+  "single_use_links_description": "Generate private URLs without exposing the username, which will be destroyed once used",
+  "add_a_single_use_link": "Add new link",
+  "single_use_link_copied": "Single use link copied",
   "invitees_can_schedule": "Invitees can schedule",
   "date_range": "Date Range",
   "calendar_days": "calendar days",
   "business_days": "business days",
   "set_address_place": "Set an address or place",
   "set_link_meeting": "Set a link to the meeting",
+  "managed_event_field_parent_control_disabled": "Can't be toggled by parent, please unlock the field to control it from child",
```

Comment:
How about this ? `This field cannot be toggled. It can only be unlocked for child event types to generate and use private links`

Reply:
Yes much better!